core/src/components/toast/toast.tsx

```diff
@@ -275,7 +290,33 @@ export class Toast implements ComponentInterface, OverlayInterface {
 
     await this.delegateController.attachViewToDom();
 
-    await present<ToastPresentOptions>(this, 'toastEnter', iosEnterAnimation, mdEnterAnimation, this.position);
+    const { position, positionAnchor } = this;
+
+    let anchor: HTMLElement | undefined;
+    if (position === 'middle' && positionAnchor !== undefined) {
+      printIonWarning('The positionAnchor property is ignored when using position="middle".', this.el);
+    } else if (typeof positionAnchor === 'string') {
+      /**
+       * If the anchor is defined as an ID, find the element.
+       * We do this on every present so the toast doesn't need
+       * to account for the surrounding DOM changing since the
+       * last time it was presented.
+       */
+
+      const foundEl = document.getElementById(positionAnchor);
+      if (foundEl === null) {
+        printIonWarning(`An anchor element with an ID of ${positionAnchor} was not found in the DOM.`, this.el);
```

Comment:
```suggestion
        printIonWarning(`An anchor element with an ID of "${positionAnchor}" was not found in the DOM.`, this.el);
```

The trigger controller wrap the ID in quotes, so we should probably do the same here: https://github.com/ionic-team/ionic-framework/blob/ac2c8e6c22da4d0d8224def24ddef56ee9d26246/core/src/utils/overlays.ts#L810